app/notification/application/map_contents.py

```diff
@@ -134,9 +160,24 @@ def format_questions_answers_with_stringIO(
         json_file = cls.get_questions_and_answers(notification)
         output = StringIO()
 
-        output.write(f"********* {Config.FUND_NAME} **********\n")
+        output.write(
+            "*********"
+            f" {Config.FUND_NAME}{' ' + ' '.join(list(json_file.keys())[0].split('cof', 1)[-1][1:].upper().split('-')).strip() if 'cof' in list(json_file.keys())[0] else ''} **********\n"  # noqa
+        )
+
         for section_name, values in json_file.items():
-            title = section_name.split("-")
+            title = (
+                [
+                    item
+                    for item in section_name.split("-")[
+                        : section_name.split("-").index("cof")
+                    ]
+                    if "cof" not in item
+                ]
+                if "cof" in section_name.split("-")
+                else section_name.split("-")
+            )
```

Comment:
We actually do not need to add the round and window title to the  {Config.FUND_NAME}, line 165, specific to cof. I was just testing it out tbh and it will not add any value I guess if the fund name changes so I am gonna remove it. 

The second list comprehension only removes the repetitive text after the section name " cof r3 w1" which is being displayed on every section for this specific round. 
Happy to huddle to discuss this?